project/Build.scala

```diff
@@ -31,6 +31,17 @@ object StorehausBuild extends Build {
       case version if version startsWith "2.10" => "2.10" // TODO: hack because sbt is broken
       case x => x
     }
+
+  def bogus210Version(dep: ModuleID) = dep cross CrossVersion.binaryMapped {
+    case version if version startsWith "2.10" => "2.10.3"
+    case x => x
+  }
+
+  def isScalaGreater29x(scalaVersion: String) = scalaVersion match {
+    case version if version startsWith "2.9" => false
+    case version if version matches "^2.1[0-2].*" => true
```

Comment:
1) I think you want: `^2\.1[0-2]\.*` here.
2) Can we use the same approach in all the new code we add? startsWith is dangerous (consider version 2.91.0 which is not the same as 2.9.x).